inc/cleantalk-settings.php

```diff
@@ -3306,7 +3306,7 @@ function apbct_settings__get__long_description()
             //HANDLE LINK
             'desc'  => sprintf(
                 esc_html__('The option helps to block bots . The honeypot field option adds a hidden field to the form. When spambots come to a website form, they can fill out each input field. Enable this option to make the protection stronger on these forms. Learn more about supported forms %s', 'cleantalk-spam-protect'),
-                '<a href="https://cleantalk.org/help/wordpress-plugin-settings{utm_mark}#honeypot" target="_blank">' . __('here.', 'cleantalk-spam-protect') . '</a>'
+                '<a href="https://cleantalk.org/help/honeypot-field{utm_mark}" target="_blank">' . __('here.', 'cleantalk-spam-protect') . '</a>'
```

Comment:
Is it possible to use link builder here?

Reply:
Yeah, it is possible, but requires refactoring here https://github.com/CleanTalk/wordpress-antispam/blob/master/inc/cleantalk-settings.php#L3367
If you think it is needed, then will be continue of task.